client/components/share/linkedin-share-preview/index.jsx

```diff
@@ -47,7 +48,11 @@ export class LinkedinSharePreview extends PureComponent {
 						{ imageUrl && (
 							<div className="linkedin-share-preview__image-wrapper">
 								<a href={ articleUrl }>
-									<img className="linkedin-share-preview__image" src={ imageUrl } />
+									<img
+										alt="Preview when shared to LinkedIn"
+										className="linkedin-share-preview__image"
+										src={ imageUrl }
+									/>
```

Comment:
@youbrokesomething I've never dealt with a11y issues before - is this an acceptable alt text for images?

Reply:
That's okay alt text. It is better to add specifics about what is being visually displayed if possible and to have unique descriptions. If there were five different images on this page and they all have the same alt text, it wouldn't be useful to the person using a screen reader.

[Here is an example taken from Mozilla:](https://moz.com/learn/seo/alt-text)
Okay: <img src="pancakes.png" alt="pancakes"> 

This alt text is only "okay" because it's not very descriptive. Yes, this is an image of a stack of pancakes. But, there's more to be said about this image.

Good: 

<img src="pancakes.png" alt="Stack of blueberry pancakes with powdered sugar">

This alt text is a better alternative because it is far more descriptive of what's in the image, while remaining concise. This isn't just a stack of "pancakes" (as the first alt text example demonstrated); it's a stack of blueberry pancakes with a dusting of powdered sugar!